### DISCUSSION TITLE
Make a postcss plugin

### DISCUSSION THREAD

**Discussion by ahabhgk** [windicss/windicss#34] [Q&A] [answered]
Hi, windicss is amazing, but I found that there is no postcss plugins like tailwindcss for me to use it in webpack or any other bundlers.

**Comment by voorjaar** [comment]
I think you can achieve this by processing the generated css.
e.g.
```js
const autoprefixer = require('autoprefixer')
const postcss = require('postcss')
const precss = require('precss')
const fs = require('fs')

fs.readFile('src/app.css', (err, css) => {
  postcss([precss, autoprefixer])
    .process(css, { from: 'src/app.css', to: 'dest/app.css' })
    .then(result => {
      fs.writeFile('dest/app.css', result.css)
      if ( result.map ) fs.writeFile('dest/app.css.map', result.map)
    })
})
```

**Comment by shiftgeist** [comment, marked as answer]
> Made a PostCSS plugin (experimental), you can give it a try, and feedback welcome :)
> https://github.com/windicss/postcss-windicss

from https://github.com/windicss/windicss/issues/184

I needed custom config path and custom entry points, found them in the source code. I think all [options](https://github.com/windicss/vite-plugin-windicss/blob/main/packages/plugin-utils/src/options.ts#L9) are passed. Here is my configuration:

```js
require('postcss-windicss')({
    config: './assets/css/tailwind.config.js',
    scan: {
        dirs: ['./layouts/', './content/'],
    },
}),
```
